tgui/packages/tgui/interfaces/CloningConsole.js

```diff
@@ -243,10 +242,10 @@ const Functions = (props, context) => {
           <Box>
             <Icon
               color={diskReadOnly ? 'bad' : 'good'}
-              name={diskReadOnly ? 'times' : 'check'}
+              name={'check'}
             />
             {' '}
-            {diskReadOnly ? 'Disk is read only.' : 'Disk is writeable.'}
```

Comment:
what's with the wording change? i thought it was clear previously

Reply:
I had crossed wires that "Disk is read only." meant "disk is unloadable", when that was in fact, what I was changing